users/migrations/0006_auto_20190722_1131.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.3 on 2019-07-22 11:31
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0005_auto_20190703_1737'),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name='customuser',
+            managers=[
```

Comment:
I guess it's okay, since the manager is now created dynamically?

Reply:
Looks like thanks to this `CustomUser` managers will no longer be tracked in migrations, so it should be ok.